app/pages/lab/apply-for-beta-form.jsx

```diff
@@ -8,28 +8,17 @@ const MINIMUM_SUBJECT_COUNT = 100;
 const REQUIRED_PAGES = ['Research', 'FAQ'];
 
 // Static functions
-const projectHasActiveWorkflows = (workflows) => {
-  return workflows.some((workflow) => {
-    return workflow.active;
-  });
+const projectHasActiveWorkflows = (project) => {
+  if (project.links.active_workflows) {
+    return true;
+  }
+  return false;
 };
 
-const projectHasMinimumActiveSubjects = (workflows) => {
-  const activeWorkflows = workflows.filter((workflow) => {
-    return workflow.active;
-  });
-  const uniqueSetIDs = activeWorkflows.reduce((sets, workflow) => {
-    return uniq(sets.concat(workflow.links.subject_sets));
-  }, []);
-  // Second parameter is an empty object to prevent request caching.
-  return apiClient.type('subject_sets', {})
-    .get(uniqueSetIDs)
-    .then((sets) => {
-      const subjectCount = sets.reduce((count, set) => {
-        return count + set.set_member_subjects_count;
-      }, 0);
-      return (subjectCount >= MINIMUM_SUBJECT_COUNT) ? true : `The project only has ${subjectCount} of ${MINIMUM_SUBJECT_COUNT} required subjects`;
-    });
+const projectHasMinimumActiveSubjects = (project) => {
+  const subjectCount = project.subjects_count;
+  return (subjectCount >= MINIMUM_SUBJECT_COUNT) ?
+    true : `The project only has ${subjectCount} of ${MINIMUM_SUBJECT_COUNT} required subjects`;
```

Comment:
Not a blocking comment, since this was in the code already, but do you know why this returns a string if false?

Reply:
No idea, sorry.